### DISCUSSION TITLE
Welcome to docker-spaceengineers Discussions!

### DISCUSSION THREAD

**Discussion by Devidian** [Devidian/docker-spaceengineers#6] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Treep3r** [comment]
Hey there!
Gotta say, you're a god to me, been trying to get space engineers running in docker for months now...
Really appreciate what you accomplished here!

**Comment by fiR3W4LL87** [comment]
Hello Devidian

I was able to create a working Docker container inside Unraid thanks to your Docker container. I was able to log in and play without any problems.

Now I don't know how to do this, to create all the configs so that others can also benefit from it in unraid, do you also know how to do this?

Kind regards

**Reply by Devidian** [reply]
Hi fiR3W4LL87,

i've never done anything with unraid, so i dont know exactly what it means to "create all the configs". Sorry! But glad to hear my docker helped you.

Best regards

**Comment by doomhanner** [comment]
Fantastic work. I don't think I have the knowledge to do it myself, but is a Torch version possible?

**Reply by Devidian** [reply]
FHi doomhanner, 

i have no experience with torch myself, so i dont know - atleast not from me. :D 

Best regards
